septop resume test failing

as seen [here](https://github.com/OpenFreeEnergy/openfe/actions/runs/28264207980/job/83747301296), the septop resuming tests are failing
```
openfe/src/openfe/tests/protocols/openmm_septop/test_septop_resume.py::TestCheckpointResuming::test_resume
```

with:
```
 E       TypeError: BaseSepTopAnalysisUnit.run() missing 5 required keyword-only arguments: 'pdb_file', 'ligand_A_indices', 'ligand_B_indices', 'smc_A', and 'smc_B'
```

Root cause is that we added structural analysis to the protocol, but didn't update the test accordingly:
https://github.com/OpenFreeEnergy/openfe/pull/1982

--- a/src/openfe/tests/protocols/openmm_septop/test_septop_resume.py
+++ b/src/openfe/tests/protocols/openmm_septop/test_septop_resume.py
@@ -8,10 +8,10 @@ import pathlib
 import shutil
 
 import gufe
+import numpy as np
 import openmm
 import pytest
 from gufe.protocols.errors import ProtocolUnitExecutionError
-import numpy as np
 from numpy.testing import assert_allclose
 from openfe_analysis.utils.multistate import _determine_position_indices
 from openff.units import unit as offunit

--- a/src/openfe/tests/protocols/openmm_septop/test_septop_resume.py
+++ b/src/openfe/tests/protocols/openmm_septop/test_septop_resume.py
@@ -11,6 +11,7 @@ import gufe
 import openmm
 import pytest
 from gufe.protocols.errors import ProtocolUnitExecutionError
+import numpy as np
 from numpy.testing import assert_allclose
 from openfe_analysis.utils.multistate import _determine_position_indices
 from openff.units import unit as offunit
@@ -233,9 +234,21 @@ class TestCheckpointResuming:
         )
 
         # Finally we analyze the results
+        # First we get some inputs
+        selection_indices = np.array(setup_results["selection_indices"])
+        ligand_A_full_indices = np.array(setup_results["ligand_A_indices"])
+        ligand_B_full_indices = np.array(setup_results["ligand_B_indices"])
+        ligand_A_indices = np.where(np.isin(selection_indices, ligand_A_full_indices))[0].tolist()
+        ligand_B_indices = np.where(np.isin(selection_indices, ligand_B_full_indices))[0].tolist()
+
         _ = analysis_unit.run(
             trajectory=sim_results["trajectory"],
             checkpoint=sim_results["checkpoint"],
+            pdb_file=setup_results["subsampled_pdb_structure"],
+            ligand_A_indices=ligand_A_indices,
+            ligand_B_indices=ligand_B_indices,
+            smc_A=analysis_unit._inputs["alchemical_components"]["stateA"][0],
+            smc_B=analysis_unit._inputs["alchemical_components"]["stateB"][0],
             scratch_basepath=tmp_path,
             shared_basepath=tmp_path,
         )

